src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx

```diff
@@ -43,9 +37,31 @@ To get the most out of the **Handled exceptions** UI, use this basic workflow:
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Exceptions > Handled exceptions:** As you explore the wealth of data in the charts and table, use groups and filters to discover patterns that help you determine the root cause of mobile app exceptions.
 </figcaption>
 
+## Features
+
+Handling exceptions as they occur can help improve your mobile app users' experience, but it's not enough to catch exceptions. You also need to know how to prevent them. Our Handled exceptions feature helps you answer these kinds of questions:
+
+* How many different types of handled exceptions are occurring? A high occurrence rate may necessitate changes to the back-end systems.
```

Comment:
It's "backend" https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#backend

Also, maybe make that sentence more active? Something like "When you're seeing a lot of handled exceptions, you may need to make changes to your backend systems"